<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDbDialog</class>
 <widget class="QDialog" name="AddDbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add an add-on</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="nameLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Enter the add-on name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="folderLayout">
     <item>
      <widget class="QRadioButton" name="folderRadio">
       <property name="text">
        <string>Select the add-on folder:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="folderButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileLayout">
     <item>
      <widget class="QRadioButton" name="fileRadio">
       <property name="text">
        <string>Select the add-on Zip file:</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="pathLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="remCheck">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Remove the add-on file/folder after addition</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filesLayout">
     <item>
      <widget class="QRadioButton" name="filesRadio">
       <property name="text">
        <string>Enter the add-on files:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filesEdit">
       <property name="placeholderText">
        <string>Seperate by comma</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
